File: Java/FinalExam2.java
import java.util.*;

/*
* Programmer: Mario Martinez
* date: 04/01/2022  
* Purpose: FinalExam2
*/

class Untitled {
    public static void main(String[] args) {
        int score = 0;
        Scanner stdIn = new Scanner(System.in);
        int count = stdIn.nextInt();
        stdIn.nextLine();
        
        String[] arr = new String[count];
        for (int i = 0; i < count; i++) {
            arr[i] = stdIn.nextLine();
            if(i > 0 && arr[i].equals(arr[i - 1])){
                score++;
            }
        }
        System.out.println(score);
        
    }
}

File: Python/FinalExam2.py
#!/usr/bin/env python3
#Programmer: MMJR ku5e@ku5e.com
#date: 04/01/2022
#Purpose: FinalExam2

score = 0
count = int(input())

grades = list()

for i in range(count):
	grades.append(input())
	if(i > 0 and grades[i] == grades[i - 1]):
		score += 1
		
print(score)

